react/store-global.css

```diff
@@ -12,13 +34,23 @@ h2,
 h3,
 h4,
 p1 {
-  font-family: Fabriga, -apple-system, BlinkMacSystemFont, avenir next, avenir,
-    helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif;
+  font-family: var(--font-family);
 }
 
 body {
-  background-color: white;
   height: 100%;
+  overflow-x: hidden;
+  margin: 0 auto 0 auto;
+  background-color: var(--color-white);
```

Comment:
It's not necessarily white.

The body tag should have the token class `bg-base`, instead.